engine/src/main/resources/assets/i18n/menu_fa.lang

```diff
@@ -0,0 +1,244 @@
+    "remove-server": "پاک کردن",
+    "reset-default": "باز نشانی",
+    "reset-fov-amount": "باز نشانی میزان میدان دید",
+    "resolution-scale": "میزان دقت",
+    "respawn": "دوباره زنده شدن بازیگر در بازی",
+    "restore-defaults": "بازگرداندن اشتباهات",
+    "return-main-menu": "خارج شدن و رفتن به فهرست اصلی",
+    "rotation-dead-zone": "چرخش محور منطقه ی مرده",
+    "save-game-path": "مسیر بازی را ذخیره کنید",
+    "screenshot-format": "فرمت اسکرین شات- قالب کردن نما گرفت",
+    "screenshot-size": "اندازه ی نما گرفت",
+    "screenshot-size-double": "اندازه ی دو برابر",
+    "screenshot-size-half": "نصف اندازه",
+    "screenshot-size-normal": "اندازه ی معمولی",
+    "screenshot-size-quarter": "یک چهارم اندازه",
+    "select-game-title": "انتخاب بازی",
+    "select-modules": "پیمانه ها",
+    "select-modules-title": "انتخاب پیمانه ها",
+    "select-multiplayer-game-sub-title": "بازیکنان متعدد",
+    "select-singleplayer-game-sub-title": "تک بازیکن",
+    "select-world-gen": "انتخاب دنیای مولد",
+    "select-world-gen-info": "با فعال سازی پیمانه ها امکان اضافه کردن بیشتر",
+    "server-address": "آدرس",
+    "server-list-complete": "لیست کارساز کامل شده",
+    "server-name": "اسم",
+    "server-owner": "مالک",
+    "server-port": "درگاه",
+    "settings-language": "زبان",
+    "settings-title": "تنظیمات",
+    "shadows": "سایه ها",
+    "shadows-off": "خاموش",
+    "shadows-on": "روشن",
+    "shadows-pcr": "روشن کردن پی. سی. آر",
+    "sound-volume": "حجم صدا",
+    "ssao": "SSAO",
+    "start-game": "بازی",
+    "start-singleplayer-game": "تک بازیکن",
+    "this-language-English": "Farsi",
+    "this-language-native": "ایرانی",
+    "update-module": "به روز رسانی",
+    "video-display-mode": "حالت صفحه ی نمایش",
+    "video-fullscreen": "صفحه ی نمایش کامل",
+    "video-windowed-fullscreen": "پنجره سازی تمام صفحه",
+    "video-windowed": "Windowed",
+    "video-graphic-settings": "پنجره ی تنظیمات گرافیکی",
+    "video-high-end-settings": "در حد زیاد-پایان  /آزمایش تنظیمات",
+    "video-preset": "از پیش نشاندن",
+    "video-preset-custom": "سفارش",
+    "video-preset-high": "بالا-زیاد",
+    "video-preset-low": "پایین-کم",
+    "video-preset-medium": "میانه",
+    "video-preset-message": "تغییر ندادن فاصله ی دید یا حوزه ی دید",
+    "video-preset-minimal": "کمترین",
+    "video-preset-ultra": "فرا",
+    "video-settings": "ویدیو",
+    "video-settings-title": "تنظیمات ویدیو",
+    "video-vsync": "V-Sync",
+    "view-distance": "فاصله ی دید",
+    "view-distance-blind": "بسته شدن قانونی",
+    "view-distance-extreme": "حداکثر",
+    "view-distance-far": "دور",
+    "view-distance-mega": "مگا",
+    "view-distance-moderate": "میانه",
+    "view-distance-near": "نزدیک",
+    "view-distance-ultra": "فرا",
+    "vignette": "تصویر",
+    "volumetric-fog": "مه حجمی",
+    "warn-modules": "اخطار: Enabling extra modules, especially all or those not in the stable lineup can cause game crashes or broken worlds.\nPlease handle with care.",
```

Comment:
This is untranslated - can you wrap this up just so the language is 100% finished? [9]